src/firefly/js/ui/CatalogSearchMethodType.css

```diff
@@ -1,32 +1,14 @@
-div.insearch {
-    height: 75px;
-    display: flex;
-    flex-direction: row;
-    justify-content: flex-end;
-    align-items: flex-end;
-}
 div.intarget {
     height: 75px;
     display: flex;
     justify-content: flex-start;
-    align-items: flex-start;
+    align-items: center;
 }
-div.size {
-    margin-top: 20px;
-    margin-bottom: 20px;
-    margin-left: 30%;
-    border-bottom: 2px solid #c8c8c8;
-    height: 60px;
+
+ul li {
+    list-style-type: none;
 }
 
-div.sizellipse {
-    height: 75px;
-    display: flex;
-    flex-direction: column;
-    justify-content: flex-start;
-    align-items: flex-start;
-/*    margin-top: 20px;
-    margin-bottom: 20px;
-    margin-left: 30%;*/
-    border: solid #a3aeb9 1px;
+li {
+    font-family: tahoma, arial, helvetica, sans-serif;
```

Comment:
this is a duplicate of our global font-family.  it is not necessary and in most cases not desirable.